src/style.css

```
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin: 0 4px 0 0;
+  background: url(images/star.svg) no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
```

1. we can simplify setting margin
2. use background `shorthand`

```suggestion
.stars__star {
  height: 16px;
  width: 16px;
  background: url(images/star.svg) no-repeat center;
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```